helm/cluster-operator-chart/templates/rbac.yaml

```diff
@@ -30,6 +30,12 @@ rules:
       - serviceaccounts
     verbs:
       - create
+  - apiGroups:
+      - "application.giantswarm.io"
+    resources:
+      - apps
+    verbs:
+      - "*"
```

Comment:
I prefer using "*" here because it's less verbose and the operator needs to manage these CRs.

But if others prefer to list the verbs I'm OK with that. @puja108 WDYT?